core/src/main/java/org/fao/geonet/kernel/url/UrlChecker.java

```diff
@@ -51,11 +59,13 @@ public Void apply(@Nullable HttpClientBuilder originalConfig) {
                     .setSocketTimeout(10000);
             RequestConfig requestConfig = config.build();
             originalConfig.setDefaultRequestConfig(requestConfig);
-            originalConfig.setUserAgent("Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/73.0.3683.86 Chrome/73.0.3683.86 Safari/537.36");
+            originalConfig.setUserAgent("GeoNetwork Link Checker");
```

Comment:
Maybe we should add a global useragent in config.properties so user can customize it if needed ?

Reply:
I've added this.